packages/jsActions/nanoflow-actions-native/src/external/DownloadFile.ts

```diff
@@ -0,0 +1,48 @@
+// This file was generated by Mendix Studio Pro.
+//
+// WARNING: Only the following code will be retained when actions are regenerated:
+// - the import list
+// - the code between BEGIN USER CODE and END USER CODE
+// - the code between BEGIN EXTRA CODE and END EXTRA CODE
+// Other code you write will be lost the next time you deploy the project.
+import fetchBlob from "rn-fetch-blob";
+
+// BEGIN EXTRA CODE
+function formatMendixFileUrl(file: mendix.lib.MxObject): string {
+    return `${mx.remoteUrl}file?guid=${file.getGuid()}&changedDate=${file.get("changedDate") ?? ""}&name=${file.get(
+        "Name"
+    )}`;
```

Comment:
I'm willing to trust the GUID to be URL-safe, but the name could be set to anything and should be URL-escaped.

Reply:
please check